<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History_Data_Query</class>
 <widget class="QMainWindow" name="History_Data_Query">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_date_type_selection">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_observe_well_selection">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>106</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_input_date">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>159</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>158</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_date_type">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间类型:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_observe_type_selection">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>132</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_table_show_query">
    <property name="geometry">
     <rect>
      <x>349</x>
      <y>159</y>
      <width>80</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>显示表查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_observe_type">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>132</y>
      <width>23</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_line_chart_show_query">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>80</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>显示图查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_oberve_well">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>106</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>观测井:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
